best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
+* #it only expects one thing (within reason).
```

Comment:
I think testing one thing, and having one expectation are two different things - I try to test one thing, but that may have multiple side-effects, so often have more than one expectation.